packages/amplify_auth_cognito/lib/src/CognitoSession/AWSCognitoUserPoolTokens.dart

```diff
@@ -18,10 +18,21 @@ import 'dart:io' show Platform;
 import 'package:amplify_auth_plugin_interface/amplify_auth_plugin_interface.dart';
 import 'package:flutter/material.dart';
 
+/// Class to wrap AWS Cognito UserPool tokens
```

Comment:
One way we can improve docs is instead of _explaining_ what the class/variable/method is from the library architecture perspective, talk about what it is from customer's use case perspective or how to use it or to document the behavior.

E,g. here the doc could be something like

```suggestion
import 'package:amplify_auth_plugin_interface/amplify_auth_plugin_interface.dart';
import 'package:flutter/material.dart';

/// Holds cognito user pool specific auth tokens. 
///
/// Returned from `Auth.fetchAuthSession()` API in the `CognitoAuthSession` object.
/// Todo: Use it to... or Use it as ... etc. if applicable.
```

Reply:
Added this on my end.

Reply:
Also I think this is a good opportunity to explain how our fetchAuthSession API works in `CognitoAuthSession.dart`:
```
/// A Cognito extension of AuthSession
/// 
/// Instantiated when fetchAuthSession is called with [getAWSCredentials] set to true.
/// If a Cognito Identity Pool is configured with unauthenticated access enabled, the 
/// [credentials] and [identityId] will be populated if fetchAuthSession is called
///  prior to user signin. 
/// If fetchAuthSession is called and an Identity Pool is not 
/// configured or is not configured with unauthenticated access enabled, then a
/// [SignedOutException] will be thrown.
```